docs/resources/Channel.md

```diff
@@ -164,7 +164,7 @@ Represents a message sent in a channel within Discord.
 | description | string | description of embed |
 | url | string | url of embed |
 | timestamp | date | timestamp of embed content |
-| color | integer | color code of the embed |
+| color | integer | is a base10 integer calculated from the following formula: `red*65536 + green*256 + blue` |
```

Comment:
also, if you change this description you must change those in https://discordapp.com/developers/docs/resources/guild to match